octoprint_mrbeam/static/js/ready_to_laser_viewmodel.js

```diff
@@ -196,6 +196,15 @@ $(function () {
                 self.fromCurrentData = function (data) {
                     self._fromData(data);
                 };
+
+                self.onDataUpdaterPluginMessage = function (plugin, data) {
+                    if (plugin !== MRBEAM.PLUGIN_IDENTIFIER) {
+                        return;
+                    }
+                    if (MRBEAM.STATE_KEY in data) {
+                        self._fromData(data, "onDataUpdaterPluginMessage");
```

Comment:
Shouldn't this one also be part of the events? 🤔 I am not sure why is this needed since in the method doesn't seem to be used or? 😕

Reply:
I added "onDataUpdaterPluginMessage" so we know where we got the state from, I would say we can leave this as a string